modules/ROOT/pages/rest-sdk/edits/customization-layer.adoc

```diff
@@ -0,0 +1,54 @@
+= Create a Customization Layer
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+After creating your connector, optionally create an instance of your existing
+connector descriptor to generate a new layer of functionality and customization
+to describe other use cases through metadata.
+
+== Customization Layer
+
+The customization layer is an instance of an existing connector descriptor.
+When you create a connector using the customization layer, the connector is
+versioned and released separately from the base connector. Using the customization
+layer enables you to develop new versions of the base connector without duplicating code.
+Changes that are made in the new version do not affect the base version of the connector,
+nor does it break backward compatibility.
+
+Using the customization layer enables you to take an immutable version of a connector
+and generate net new assets that add new layers of functionality. These newly
+generated composite artifacts can have their own release lifecycle and be developed
+separately from the base connector.
```

Comment:
"net new assets"? = results from new use cases?
"composite artifacts"? = versions of base connector?
This para sounds a bit jargony to me.